happi/client.py

```diff
@@ -325,6 +326,84 @@ def load_device(self, use_cache=True, **post):
         cntr = self.find_device(**post)
         return from_container(cntr, use_cache=use_cache)
 
+    def change_container(
+        self,
+        item: HappiItem,
+        target: Type[HappiItem],
+        edits: Dict[str, Any] = {},
+        how: str = 'right'
+    ) -> Dict[str, Any]:
+        """
+        Takes an instance of one item (device) and transfers its contents
+        into a new ``target`` container.  Checks are performed to ensure
+        the contents are compatible (follwing enforce requirements in the
+        target container)
+
+        This function is built to be a helper function for an interactive
+        cli tool, and passes information up accordingly.  If keys are not
+        significantly mismatched, this function can be used as is.
+
+        Parameters
+        ----------
+        item : happi.HappiItem
+            HappiItem instance to be transferred to a new container
+        target : Type[happi.HappiItem]
+            Container to contain new item
+        edits : dict[str, Any], optional
+            Dictionary of edits to supercede values in original item
+        how : str, optional
+            Method of resolving the entries between the original item
+            and target container.  Can be:
+            - right : Expect a value for every entry in target container
+            - inner : Expect values for only entries in BOTH original
+            item and target container
+
+        Raises
+        ------
+        TransferError
+            If there is a problem tranferring item into container
+
+        Returns
+        -------
+        new_kwargs : Dict[str, Any]
+            kwargs necessary to load a device
```

Comment:
Are these the kwargs we need to pass into the new happi item? Using `device` at all in the docstring anywhere is instantly confusing for me.

Reply:
I was also confused for a while between items and devices, particularly with the Device deprication... 😬